app/assets/javascripts/discourse/app/models/topic.js

```diff
@@ -837,8 +837,10 @@ Topic.reopenClass({
     return ajax(`/t/id_for/${slug}`);
   },
 
-  setSlowMode(topicId, seconds) {
+  setSlowMode(topicId, seconds, enabledUntil) {
     const data = { seconds };
+    data["enabled_until"] = enabledUntil;
```

Comment:
```suggestion
    data.enabled_until = enabledUntil;
```
We shouldn't use quotes on names unless required.